### PR TITLE
feat(QoL): remove colour prefixes from server name when updating console title

### PR DESCRIPTION
**Pull request by Lierrmm**
Quality of Life improvement that removes coloured prefixes from server name when updating console title.

Old:
image

New:
image

### CODE REVIEW

File: src/client/component/dedicated_info.cpp [hunk 1]
```diff
@@ -18,16 +19,20 @@ namespace dedicated_info
 			const auto sv_running = game::Dvar_FindVar("sv_running");
 			const auto server_name = game::get_dvar_string("live_steam_server_name");
 
+			// QoL improvement - Remove colour codes from server name for console title
```
Comment (momo5502) [inline src/client/component/dedicated_info.cpp:22]: please remove the comment

image

File: src/client/component/dedicated_info.cpp [hunk 2]
```diff
@@ -18,16 +19,20 @@ namespace dedicated_info
...
+			std::regex vowel_re("\\^[0-9;:]");
+			const auto clean_server_name = std::regex_replace(server_name, vowel_re, "");
```
Comment (momo5502) [inline src/client/component/dedicated_info.cpp:24]: the game provides some I_CleanStr function which does exactly that but in a way that is guaranteed to clean the string in a compatible way, please use that instead

**Review by momo5502 — CHANGES_REQUESTED** [review]

**Comment by momo5502** [comment]
no need to close it, it was a good pr, if you strugglenwith the i cleanstr thing, i can help you

**Comment by Lierrmm** [comment]
Had some trouble with some git merges so will re-open once I've resolved those.

**Pull request closed without merging**